app/controllers/pictures_controller.rb

```diff
@@ -0,0 +1,94 @@
+class PicturesController < ApplicationController
+  
+  def index
+    if params[:user_id].present?
+      if current_user.id == params[:user_id]
+        @user = current_user
+      else
+        @user = User.find(params[:user_id])
+      end
+      @pictures = @user.pictures.all
```

Comment:
@ianroberts131 The `.all` can be removed on lines 10 and 13, it will still work as intended.